int32 id
string status
---
int8 errorcode
string errormsg
